distribution/pom.xml

```diff
@@ -171,7 +171,6 @@
 
     <modules>
         <module>fully-loaded</module>
```

Comment:
> I kept it for elasticsearch java client users because they have to do all those imports by themselves.

Actually, they will be able to depend on:

```
        <dependency>
            <groupId>org.elasticsearch.distribution</groupId>
            <artifactId>distributions</artifactId>
            <version>${elasticsearch.version}</version>
            <type>pom</type>
        </dependency>
```